packages/nextjs/components/RandomLoadingBackground.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Jazzicon, { jsNumberForAddress } from "react-jazzicon";
+import { cn } from "~~/utils/cn";
+
+// Addresses to cycle through
+const ADDRESSES = [
+  "0x815E0023A28AA56ae60148B3781D783b223953B6",
+  "0xa63Bfd6492Ed2098e417fe2A7dE30d39d9CbC102",
+  "0xb5C23229fB8462c1658F40d97a11D94feB57e223",
+  "0x9E9be8440794f28Ac08e13ec290B688520a5D824",
+];
```

Comment:
I was going to generate random addresses, but the effect goes by so fast that I don't think it's worth it. Just having a list of addresses works good enough from what I can see, and you can somewhat control the different colors that show up.